.github/workflows/win.yml

```diff
@@ -111,14 +109,26 @@ jobs:
             -DENABLE_CCACHE=OFF \
             -DENABLE_SERVICES=OFF \
             -DENABLE_BENCHMARKS=OFF \
-            -DPREFER_EXTERNAL=ON \
+            -DPREFER_EXTERNAL=ON
+
+      - if: ${{ steps.vcpkg-restore.outputs.cache-hit != 'true' }}
+        name: Save vcpkg packages (if cache miss)
+        uses: actions/cache/save@v3
+        with:
+          key: ${{ steps.vcpkg-restore.outputs.cache-primary-key }}
+          path: |
+            vcpkg/*
```

Comment:
also `vcpkg/*` is a notation that's needed when you want to exclude subdirectories, it's basically a bug in `action/cache`